src/jimgw/prior.py

```diff
@@ -143,6 +144,68 @@ def log_prob(self, z: dict[str, Float]) -> Float:
         return -0.5 * variable**2 - 0.5 * jnp.log(2 * jnp.pi)
 
 
+@jaxtyped(typechecker=typechecker)
+class SimplexBaseDistribution(Prior):
```

Comment:
Can this not be generalized to higher dimensions? I am pretty sure the definition of a simplex distribution is general.